rwengine/src/ai/CharacterController.cpp

```diff
@@ -174,23 +174,24 @@ void CharacterController::steerTo(const glm::vec3 &target) {
     float steeringAngle = 0.0f;
     float deviation = glm::abs(vehicle->isOnSide(target)) / 5.f;
 
-    // If we are almost at the right angle, decrease the deviation to reduce wiggling
-    if (deviation < 1.f) deviation = deviation / 5.f;
+    // If we are almost at the right angle, decrease the deviation to reduce
+    // wiggling
+    if (deviation < 1.f)
+        deviation = deviation / 5.f;
 
     // Make sure to normalize the value
     deviation = glm::clamp(deviation, 0.f, 1.f);
 
     // Set the right sign
     steeringAngle = std::copysign(deviation, -vehicle->isOnSide(target));
 
-    vehicle->setSteeringAngle(steeringAngle, true);  
+    vehicle->setSteeringAngle(steeringAngle, true);
 }
 
 // @todo replace this by raytest/raycast logic
-bool CharacterController::checkForObstacles()
-{
+bool CharacterController::checkForObstacles() {
     // We can't drive without a vehicle
-    VehicleObject* vehicle = character->getCurrentVehicle();
+    VehicleObject *vehicle = character->getCurrentVehicle();
```

Comment:
So for references ampersand is part of the type, but for pointers, it is part of the variable name???

Reply:
Yes, I had PointerAlignment set to the wrong type. It should be uniform to the left now.